docs/checks/internal/checks.yaml

```diff
@@ -447,17 +447,18 @@ checks:
     risk: Medium
     tags: supply-chain, security, dependencies
     repos: GitHub, local
-    short: Determines if the project has declared and pinned its dependencies.
+    short: Determines if the project has declared and pinned the dependencies of its build process.
     description: |
       Risk: `Medium` (possible compromised dependencies)
 
-      This check tries to determine if the project pins its dependencies.
+      This check tries to determine if the project pins dependencies used during its build and release process.
       A "pinned dependency" is a dependency that is explicitly set to a specific hash instead of
```

Comment:
Can you squeeze in a link to https://github.com/ossf/package-manager-best-practices/blob/main/published/npm.md for npm?

Reply:
I think that actually would increase the confusion as it dives deep into package-lock.json, yarn.lock, etc and that's where library maintainers are currently feeling that this check is not relevant for them.

Reply:
Resolving for now. Can create separate PR if needed.